docs/application-setup.md

```diff
@@ -61,18 +83,48 @@ $ live-server --entry-file=index.html public
 
 1. If you do not have the Angular CLI installed, download and install it from [this URL](https://cli.angular.io/).
 
-2. Use the following commands to create your application, install Luigi, make assets available, and serve your application.
+2. Use the following installer to create your application, install Luigi, make assets available, and serve your application.
 
 ````
-$ ng new my-dream-app --routing && cd my-dream-app
-$ npm i -S @kyma-project/luigi-core@latest @kyma-project/luigi-client@latest fiori-fundamentals@latest
-$ mv src/index.html src/angular.html
-$ curl https://raw.githubusercontent.com/kyma-project/Luigi/master/core/examples/luigi-sample-angular/src/index.html > src/index.html
-$ curl https://raw.githubusercontent.com/kyma-project/Luigi/master/core/examples/luigi-sample-angular/src/assets/basicConfiguration.js > src/assets/sampleconfig.js
-$ sed 's/"src\/index.html"/"src\/angular.html"/g' angular.json > tmp.json && mv tmp.json angular.json
-$ sed 's/"src\/assets"/"src\/assets","src\/index.html", {"glob": "**","input": "node_modules\/@kyma-project\/luigi-core", "output": "\/luigi-core"},{"glob": "**","input": "node_modules\/@kyma-project\/luigi-client","output": "\/luigi-client"}/g' angular.json > tmp.json && mv tmp.json angular.json
-$ ng serve
+bash <(curl -s https://raw.githubusercontent.com/kyma-project/Luigi/master/scripts/setup/angular.sh)
 ````
+or execute these commands manually to get the same result:
+```bash
+ng new my-angular-app --routing && cd my-angular-app
+
+npm i 
```

Comment:
this is probably not needed, as `ng new` above performs npm packages installation